editoast/src/views/infra/mod.rs

```diff
@@ -101,6 +101,9 @@ pub enum InfraApiError {
     #[error("Infra '{infra_id}', could not be found")]
     #[editoast_error(status = 404)]
     NotFound { infra_id: i64 },
+    #[error("Validation errors on speed section(s) : '{}'", .speed_sections.join(", "))]
+    #[editoast_error(status = 422)]
+    SpeedSectionValidationError { speed_sections: Vec<String> },
```

Comment:
We don't need this. Validation errors can be transformed automatically into `EditoastError` (check `errors.rs`)